src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -147,11 +147,17 @@ private string AddActionsAndGetTitle(State state, ArrayBuilder<CodeAction> actio
 
                 if (!BlockOverlapsHiddenPosition(block, cancellationToken))
                 {
-                    actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+                    actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false, includeLValues: true));
 
                     if (blocks.All(b => !BlockOverlapsHiddenPosition(b, cancellationToken)))
                     {
-                        actions.Add(CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+                        var localForAllCodeAction = CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false, includeLValues: false);
+                        actions.Add(localForAllCodeAction);
```

Comment:
local variable seems unused.  just inline this to match the rest of the code surrounding this?